packages/documentation-site/patternfly-docs/content/design-guidelines/styles/motion/motion.md

```diff
@@ -0,0 +1,154 @@
+</p>
+</Alert>
+<br />
+
+Like color or typography, **motion** can create a strong foundation that helps connect the complex elements within your designs. By carefully incorporating motion into the design of a UI, you can create products that are more clear, engaging, and expressive.  Effective motion design should be intentional, and should be used to help guide users through their experiences more efficiently.
+
+This guide outlines the principles that drive motion design in PatternFly and describes the different types of motion that are supported in our design system.
+
+## Principles 
+These principles guide the use of motion in PatternFly, to ensure consistency and alignment with our [brand voice and tone](https://www.patternfly.org/ux-writing/brand-voice-and-tone).
+
+<br />
+
+<Grid hasGutter>
+<GridItem span={6}>
+    <Card isFullHeight>
+    <CardHeader>
+    <CardTitle>Intentional</CardTitle>
+    </CardHeader>
+    <CardBody>Motion should be intentionally designed in ways that help your users reach their goals. Animations should be quick and simple to focus your users’ attention only on what’s important, without distracting.
+    </CardBody>
+    </Card>
+</GridItem>
+<GridItem span={6}>
+    <Card isFullHeight>
+    <CardHeader>
+    <CardTitle>Consistent</CardTitle>
+    </CardHeader>
+    <CardBody>Motion should be applied consistently to ensure that you’re sticking to the best design practices. Consistent animations provide a sense of familiarity to users throughout their journey and connect cross-product experiences. 
+    </CardBody>
+    </Card>
+</GridItem>
+<GridItem span={6}>
+    <Card isFullHeight>
+    <CardHeader>
+    <CardTitle>Engaging</CardTitle>
+    </CardHeader>
+    <CardBody>To draw your users in and keep them interested, motion should be expressive and approachable. Animations should celebrate the big moments in our users’ journeys and add character to common interactions. 
+    </CardBody>
+    </Card>
+</GridItem>
+<GridItem span={6}>
+    <Card isFullHeight>
+    <CardHeader>
+    <CardTitle>Inclusive</CardTitle>
+    </CardHeader>
+    <CardBody>
+        Motion should contribute to experiences that offer all of your users a sense of belonging. Animations should always be optional and should respect your users’ motion preferences.
+    </CardBody>
+    </Card>
+</GridItem>
+</Grid>
+
+## Motion in PatternFly
+
+PatternFly components support motion in the form of **micro animations**, which are small, unobtrusive, and often understated. These animations serve specific purposes within a UI, and are designed to be subtle and complementary. 
+
+This behavior is built into PatternFly, so there is nothing additional you need to do to support motion. To meet accessibility requirements, reduced motion support is determined based on users' browser settings.
```

Comment:
This isn't universally true. Some things are built in - like the background hover transition on a button. Other things may need to be added ad hoc, like applying a rotation to a cog icon. We'd supply the class that implements the rotation, but the consumer might need to specifically add it. It's a little TBD still :-) 

For the second sentence, I think I'd say something along the lines of "Wherever PatternFly provides animation, reduced motion support (based on user's browser settings) is provided."
(which isn't quite true yet with those that existed in pf5)